[DOC-1088] Docs for Voyager v2025.11.2 (#29449)
yugabyte/yugabyte-db: docs/content/stable/yugabyte-voyager/release-notes.md

Voyager releases (starting with v2025.5.2) use the numbering format `YYYY.M.N`, where `YYYY` is the release year, `M` is the month, and `N` is the number of the release in that month.

## v2025.11.2 - November 25, 2025

### New feature

- Assess migration now supports fetching metadata from PostgreSQL read replicas in addition to the primary source database. This provides a more comprehensive and accurate workload assessment by aggregating metrics across both primary and replica nodes.

### Enhancement

- Added sorting functionality to the "Object Usage" column in the Performance Optimizations table of the assessment report for a better user experience.

### Bug fixes

- Fixed a nil pointer error when a table was missing in the target database during live migration.

> - Fixed a bug in the Schema Optimization Report to exclude parent partitioned tables from "Colocated" recommendations, as colocation applies only to leaf partitions.

Reviewer: ```suggestion
- Schema Optimization Report now excludes parent partitioned tables from "Colocated" recommendations, as colocation applies only to leaf partitions.
```
